src/utils/buildGridStyle.js

```diff
@@ -0,0 +1,10 @@
+const getStartXs = (time, { cells }) => cells.map(cell => time.toX(cell.start))
+
+const buildGridStyle = (time, timebar) => {
+  const gradient = getStartXs(time, timebar).reduce((str, x) =>
+    `${str} white ${x}px, #eee ${x}px, #eee ${x + 1}px, white ${x + 1}px,`
+  , '').slice(0, -1)
+  return { background: `linear-gradient(to right, ${gradient})` }
```

Comment:
Not sure about doing this as `linear-gradient` background - not sure if better to have DOM elements (as per timebar cells).